packages/web3-eth-iban/src/index.ts

```diff
@@ -79,7 +83,7 @@ export class Iban {
 			const paddedBigInt = leftPad(parsedBigInt, 40);
 			return toChecksumAddress(paddedBigInt);
 		}
-		throw new IbanLengthError();
+		throw new Error('Iban is indirect and cannot be converted. Must be length of 34 or 35');
```

Comment:
For Removing circular dependency https://github.com/ChainSafe/web3.js/pull/4742/commits/6b752ba74246d270a2b6e177b234e4ae69d9e5b2